package tps.proto;
message RtTpsProtoPatient
{
	required string patientUID		= 1;
	optional string patientNamePrefix = 2;
	optional string patientFirstName = 3;
	optional string patientMiddleName = 4;
	optional string patientLastName = 5;
	optional string patientNameSuffix = 6;
	optional string patientID		= 7;
	optional int32  patientsex		= 8;
	optional string patientDateOfBirth = 9;
	optional bool isinternal = 10;
}